frontend/src/routes/Infrastructure/Clusters/ClusterSets/components/EditSubmarinerConfigModal.tsx

```diff
@@ -81,6 +84,14 @@ export function EditSubmarinerConfigModal(props: EditSubmarinerConfigModalProps)
                             value={nattPort}
                             onChange={(port) => setNattPort(port)}
                         />
+                        <AcmSelect
+                            id="natt-enable"
+                            label={t('submariner.install.form.nattenable')}
+                            placeholder={t('submariner.install.form.nattenable.placeholder')}
+                            labelHelp={t('submariner.install.form.nattenable.labelHelp')}
+                            value={nattEnable}
+                            onChange={(enable) => setNattEnable(enable)}
+                        />
```

Comment:
This select does not have options which makes it fail tests.
Either add options for true/false or better yet make this a checkbox.